src/System.Drawing.Common/tests/mono/System.Imaging/MetafileTest.cs

```diff
@@ -613,6 +499,15 @@ public void CreateFilename_MultipleGraphics_EmfPlusOnly()
         [ConditionalFact(Helpers.GdiplusIsAvailable)]
         public void Measure()
         {
+            Font test_font = null;
+            try
+            {
+                test_font = new Font(FontFamily.GenericMonospace, 12);
+            }
+            catch (ArgumentException)
```

Comment:
Should we fail the test or exit out if we can't create the font? Seems like we could be in for a null-ref or some other problem if we continue on after an exception.

Reply:
Makes sens. I guess the try/catch was there historically because this code was in the constructor; catching the exception would let the other tests succeed.

I removed the try/catch alltogether - if the font is not available, this test will now fail with an ArgumentException instead of a NullReferenceException.